refarch-integrations/refarch-email-integration/refarch-email-integration-core/src/main/java/de/muenchen/refarch/email/integration/domain/exception/LoadAttachmentError.java

```diff
@@ -1,7 +1,16 @@
 package de.muenchen.refarch.email.integration.domain.exception;
 
-public class LoadAttachmentError extends Error {
+import java.io.Serial;
+
+public class LoadAttachmentError extends RuntimeException {
+    @Serial
+    private static final long serialVersionUID = 1L;
```

Comment:
We had a simiilar issue in refarch-templates repository. I think it would be better to suppress the warning for missing serial ID here (instead of adding it) as we are not (de)serializing Exception classes.

Reply:
Done